symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkSSLConfig.java

```diff
@@ -0,0 +1,14 @@
+package com.symphony.bdk.core.config.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BdkSSLConfig {
+
+    private String truststorePath;
```

Comment:
trustStorePath maybe?

Reply:
in V1, it was defined as "truststorePath". And I also think "truststore" is a complete word.
https://docs.oracle.com/cd/E19830-01/819-4712/ablqw/index.html

Reply:
+1 : `trustStorePath` and `trustStorePassword` (cf. Oracle doc : https://docs.oracle.com/cd/E19509-01/820-3503/6nf1il6er/index.html)